### PR TITLE
Some clean up, added Nginx service monitored by Consul.

### PR DESCRIPTION
**Pull request by greenbrian**

### CODE REVIEW

File: modules/compute/bootstrap.sh.tpl [hunk 1]
```diff
@@ -47,7 +47,7 @@ echo "Consul installation complete."
 
 # Get VM private ip address
 INSTANCE_PRIVATE_IP=$(ifconfig eth0 | grep "inet addr" | awk '{ print substr($2,6) }')
-
+INSTANCE_PUBLIC_IP=$(dig +short myip.opendns.com @resolver1.opendns.com)
```
Comment (tdsacilowski) [inline modules/compute/bootstrap.sh.tpl]: I'm already passing in `var.public_ip` to the `template_file` resource and passing that to the template as `join_ip = "${var.public_ip[0]}"`. Do you prefer for me to use your method or my method?

File: modules/resource_group/resource_group.tf [hunk 2]
```diff
@@ -2,3 +2,43 @@ resource "azurerm_resource_group" "demo" {
   name     = "${var.name}"
   location = "${var.location}"
 }
+
+resource "azurerm_network_security_group" "demo-west" {
+  name                = "consul-west"
+  location            = "West US"
+  resource_group_name = "${azurerm_resource_group.demo.name}"
+}
```
Comment (tdsacilowski) [inline modules/resource_group/resource_group.tf]: Should this be refactored to include all 3 datacenters and maybe be moved to the networking module?

**Review by tdsacilowski — CHANGES_REQUESTED** [review]

File: modules/compute/bootstrap.sh.tpl (hunk 1 above)
Comment (greenbrian) [reply modules/compute/bootstrap.sh.tpl]: artifact from my hack. i'll yank it.

File: modules/resource_group/resource_group.tf (hunk 2 above)
Comment (greenbrian) [reply modules/resource_group/resource_group.tf]: not required. we're already creating vnets in other regions. removing the offending code.

**Pull request merged**